.github/workflows/e2e.yaml

```diff
@@ -14,6 +14,8 @@ jobs:
       fail-fast: true
     runs-on: ${{ matrix.os }}
     name: ${{ matrix.os }} | Node ${{ matrix.node-version }} latest
+    env:
```

Comment:
Not sure about this one Rob, if our tests don't pass with telemetry on... it might crash on user's CI too

Reply:
We should discuss with @thedavidprice... as it stands the e2e test spams the database with all of the setup commands for the test app. He mentioned disabling for our CI, but maybe we just add a field to the db like `isRedwoodCI` and ignore them in reporting?

Reply:
Yeah, I think it would be helpful to filter out on the other side, because otherwise the e2e tests are ignoring a crucial aspect of ci!